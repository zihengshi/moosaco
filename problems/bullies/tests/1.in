2
4 2 5
1 1 1 1
1 3 3 3
1 3 7 3
1 3 3 3
4 2 5
1 1 1 1
1 1 3 3
1 3 7 3
1 3 3 3
