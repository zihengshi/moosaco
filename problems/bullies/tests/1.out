you're safe this time bucko
uh oh!
